package.json

```diff
@@ -51,7 +51,26 @@
     "test-cov-ci": "echo",
     "eject": "react-scripts eject",
     "prepare": "husky install",
+<<<<<<< HEAD
+    "switch-state": "npm run terra:init -target=\"$npm_config_target\" -- -reconfigure",
+    "switch-state:dev": "npm run switch-state -target=development",
+    "switch-state:stg": "npm run switch-state -target=staging",
+    "switch-state:prod": "npm run switch-state -target=production",
+    "terra:init": "cd terraform && terraform init -backend-config=\"environment/$npm_config_target.backend.conf\"",
+    "terra:init-wrks": "cd terraform && if ! terraform workspace list | grep -q \"\\s$npm_config_name$\"; then terraform workspace new \"$npm_config_name\"; fi",
```

Comment:
Is `terra:init-wrks` needed here? Was this added to test?

Reply:
yeah and for the pre-provision